.github/workflows/unit-tests-workflow.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.5.0'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.6.0'
```

Comment:
Same as above; you'll want to change this from `2.6.0` to `2.6`.

Reply:
Why did we use tags then in the first place?

Reply:
The tags are helpful when OSD bumps the version on their branches. Often times, they're release branches will be set to an incremental version which doesn't align with what we have on that same release branch.